composer/utils/checkpoint.py

```diff
@@ -321,23 +319,20 @@ def download_checkpoint(path: str,
                     pass
 
     finally:
-        # Wait for all checkpoints on the node to finish downloading
```

Comment:
why is this comment removed? can you add a shorter one describing what we're doing here

Reply:
Yep, I moved it closer to the logic